### DISCUSSION TITLE
To run multiple OpenAI-compatible end points in the same front-end.

### DISCUSSION THREAD

**Discussion by thusinh1969** [open-webui/open-webui#2200] [General]
**Is your feature request related to a problem? Please describe.**
**GGUF posed accuracy drop because of missing of bfloat16.  We have to switch to vLLM and Triton.** 

**Describe the solution you'd like**
**Wishes to be able to run multiple OpenAI-compatible end points in the same front-end of Open WebUI likes what it does with ollama's multiple GGUF.** 

**Additional context**
This will be great as Open WebUI is one of the best for simplification, pre-prompting and all.

Thanks,
Steve

**Comment by justinh-rahb** [comment]
https://docs.openwebui.com/tutorial/openai
